assets/airports/kchs.json

```diff
@@ -0,0 +1,10037 @@
+        "TYLOR": ["N32d52m14.612", "W80d11m27.582"],
+        "UNJAM": ["N34d12m56.552", "W80d48m10.587"],
+        "UZADY": ["N32d46m18.656", "W80d02m47.093"],
+        "VAN":   ["N33d28m29.355", "W80d26m54.449", "Vance"],
+        "WANDO": ["N33d02m03.469", "W79d50m26.905"],
+        "WBODY": ["N34d08m54.495", "W81d37m54.319"],
+        "WILIT": ["N33d29m30.797", "W79d27m38.471"],
+        "WOGIB": ["N33d37m59.033", "W82d18m43.983"],
+        "ZUNSU": ["N32d49m33.581", "W80d04m36.975"]
+    },
+    "runways": [
+        {
+            "name": ["3", "21"],
+            "end": [
+                ["N32d53m05.173", "W80d02m43.914", "45ft"],
+                ["N32d54m08.270", "W80d02m10.208", "45ft"]
+            ],
+            "ils": [false, false]
+        },
+        {
+            "name": ["15", "33"],
+            "end": [
+                ["N32d54m46.761", "W80d02m53.168", "45ft"],
+                ["N32d53m31.386", "W80d01m56.156", "45ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "LGRHD3": {
+            "icao": "LGRHD3",
+            "name": "Logger head Three",
+            "altitude": 4000,
+            "rwy": {
+                "KCHS3": ["HALEE", "LGRHD", "GIPPL"],
+                "KCHS15": ["LGRHD", "GIPPL"],
+                "KCHS21": ["LGRHD", "GIPPL"],
+                "KCHS33": ["HALEE", "LGRHD", "GIPPL"]
+            },
+            "exitPoints": {
+                "GIPPL": ["GIPPL"]
+            },
+            "draw": [
+                ["HALEE", "LGRHD", "GIPPL"],
+                ["LGRHD", "GIPPL"]
+            ]
+        },
+        "MLTRE2": {
+            "icao": "MLTRE2",
+            "name": "Molter Two",
+            "altitude": 4000,
+            "rwy": {
+                "KCHS3": ["MLTRE"],
+                "KCHS15": ["ANGRE", "MLTRE"],
+                "KCHS21": ["ANGRE", "MLTRE"],
+                "KCHS33": ["MLTRE"]
+            },
+            "exitPoints": {
+                "JYMSE": ["JYMSE"],
+                "VAN":   ["VAN"]
+            },
+            "draw": [
+                ["ANGRE", "MLTRE"],
+                ["MLTRE", "JYMSE*"],
+                ["MLTRE", "VAN*"]
+            ]
+        },
+        "PLMTO3": {
+            "icao": "PLMTO3",
+            "name": "Plumtree Three",
+            "altitude": 4000,
+            "rwy": {
+                "KCHS3": ["LOIRN", "PLMTO"],
+                "KCHS15": ["JAYLN", "PLMTO"],
+                "KCHS21": ["JAYLN", "PLMTO"],
+                "KCHS33": ["LOIRN", "PLMTO"]
+            },
+            "exitPoints": {
+                "CRE":  ["CRE"]
+            },
+            "draw": [
+                ["LOIRN", "PLMTO"],
+                ["JAYLN", "PLMTO"],
+                ["PLMTO", "CRE*"]
+            ]
+        },
+        "STUNO1": {
+            "icao": "STUNO1",
+            "name": "Stuno One",
+            "altitude": 4000,
+            "rwy": {
+                "KCHS3": [ "BRYSE", "STUNO"],
```

Comment:
Some waypoints missing here. GULLA and STUNO should be moved into the body. GECHE is missing from 15 and 21.